app-misc/broot/broot-1.7.0.ebuild

```diff
@@ -0,0 +1,267 @@
+hashlink-0.6.0
+heck-0.3.3
+hermit-abi-0.1.19
+id-arena-2.2.1
+idna-0.2.3
+image-0.23.14
+indexmap-1.7.0
+instant-0.1.10
+is_executable-1.0.1
+itoa-0.4.7
+jobserver-0.1.22
+jpeg-decoder-0.1.22
+lazy-regex-2.2.2
+lazy-regex-proc_macros-2.2.2
+lazy_static-1.4.0
+lazycell-1.3.0
+lfs-core-0.4.2
+libc-0.2.98
+libgit2-sys-0.12.21+1.1.0
+libsqlite3-sys-0.20.1
+libz-sys-1.1.3
+line-wrap-0.1.1
+linked-hash-map-0.5.4
+lock_api-0.4.4
+log-0.4.14
+matches-0.1.8
+memchr-2.4.0
+memmap-0.7.0
+memoffset-0.6.4
+minimad-0.7.1
+minimad-0.9.0
+miniz_oxide-0.3.7
+miniz_oxide-0.4.4
+mio-0.7.13
+miow-0.3.7
+ntapi-0.3.6
+num-integer-0.1.44
+num-iter-0.1.42
+num-rational-0.3.2
+num-traits-0.2.14
+num_cpus-1.13.0
+once_cell-1.8.0
+onig-6.2.0
+onig_sys-69.7.0
+open-1.7.1
+open-2.0.0
+parking_lot-0.11.1
+parking_lot_core-0.8.3
+pathdiff-0.2.0
+percent-encoding-2.1.0
+phf-0.9.0
+phf_generator-0.9.0
+phf_macros-0.9.0
+phf_shared-0.9.0
+pkg-config-0.3.19
+plist-1.2.0
+png-0.16.8
+ppv-lite86-0.2.10
+proc-macro-hack-0.5.19
+proc-macro2-1.0.28
+proc-status-0.1.1
+quick-xml-0.22.0
+quote-1.0.9
+rand-0.8.4
+rand_chacha-0.3.1
+rand_core-0.6.3
+rand_hc-0.3.1
+rayon-1.5.1
+rayon-core-1.9.1
+redox_syscall-0.2.9
+redox_users-0.4.0
+regex-1.5.4
+regex-automata-0.1.10
+regex-syntax-0.6.25
+remove_dir_all-0.5.3
+rusqlite-0.24.2
+ryu-1.0.5
+safemem-0.3.3
+same-file-1.0.6
+scoped_threadpool-0.1.9
+scopeguard-1.1.0
+secular-1.0.1
+serde-1.0.127
+serde_derive-1.0.127
+serde_json-1.0.66
+signal-hook-0.1.17
+signal-hook-0.3.9
+signal-hook-mio-0.2.1
+signal-hook-registry-1.4.0
+siphasher-0.3.6
+smallvec-1.6.1
+splitty-0.1.0
+str-buf-1.0.5
+strict-0.1.4
+strsim-0.8.0
+svg-0.8.2
+syn-1.0.74
+syntect-4.6.0
+tempfile-3.2.0
+termimad-0.10.3
+termimad-0.17.0
+terminal-clipboard-0.3.0
+termux-clipboard-0.1.0
+textwrap-0.11.0
+thiserror-1.0.26
+thiserror-impl-1.0.26
+tiff-0.6.1
+time-0.1.43
+tinyvec-1.3.1
+tinyvec_macros-0.1.0
+toml-0.5.8
+umask-1.0.0
+unicode-bidi-0.3.5
+unicode-normalization-0.1.19
+unicode-segmentation-1.8.0
+unicode-width-0.1.8
+unicode-xid-0.2.2
+url-2.2.2
+users-0.11.0
+vcpkg-0.2.15
+vec_map-0.8.2
+version_check-0.9.3
+walkdir-2.3.2
+wasi-0.10.2+wasi-snapshot-preview1
+weezl-0.1.5
+winapi-0.3.9
+winapi-i686-pc-windows-gnu-0.4.0
+winapi-util-0.1.5
+winapi-x86_64-pc-windows-gnu-0.4.0
+x11-clipboard-0.5.3
+xcb-0.10.1
+xml-rs-0.8.4
+yaml-rust-0.4.5
+"
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit bash-completion-r1 cargo python-any-r1
+
+DESCRIPTION="A new way to see and navigate directory trees"
+HOMEPAGE="https://dystroy.org/broot/ https://github.com/Canop/broot"
+SRC_URI="https://github.com/Canop/broot/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	$(cargo_crate_uris)"
+
+LICENSE="Apache-2.0 BSD-2 BSD LGPL-3+ MIT ZLIB"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="X"
+
+RDEPEND="
+	dev-libs/libgit2:=
+	X? ( x11-libs/libxcb:= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="X? ( ${PYTHON_DEPS} )"
+
+QA_FLAGS_IGNORED="usr/bin/broot"
+
+pkg_setup() {
+	use X && python-any-r1_pkg_setup # Used by XCB crate
```

Comment:
Well, so much for this... since the xcb crate was updated, I had a look at it... they removed the python script :eyes:
```
## [0.10.0] - 2021-09-19 - rtbo
### Changed
- build script is written in Rust. Python no longer needed. (#56, #62, #89, #99)
```
So all the python cruft can be removed (inherit, compat, pkg_setup, BDEPEND), builds fine even if python isn't found now.

Reply:
Neat. Pushed a new version with the Python stuff removed.

Broot's changelog also says "fix clipboard features not working on some recent linux distributions", and that functionality now works for me as well.